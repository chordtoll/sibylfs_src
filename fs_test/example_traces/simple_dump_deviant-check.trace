# processing file 'example_traces/simple_dump-int.trace' ...
        @type trace
     1: open_close "rebel_zero" [O_CREAT] <rw------->
        Tau
        RV_none
     2: dump-result /
           "/"|D|0
           "/rebel_zero"|F|0|0|"0"
        end dump-result
