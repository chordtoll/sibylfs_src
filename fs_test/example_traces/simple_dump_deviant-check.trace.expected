# processing file 'example_traces/simple_dump_deviant-check.trace' ...
        # processing file 'example_traces/simple_dump-int.trace' ...
        @type trace
     3: open_close "rebel_zero" [O_RDONLY;O_CREAT] 0o600
     4: Tau
     5: RV_none
     6: dump-result "/"
           "/"|D|0
           "/rebel_zero"|F|0|0|"0"
        end dump-result
# 
#         "/"|D|0
#         "/rebel_zero"|F|1|0|"da39a3ee5e6b4b0d3255bfef95601890afd80709"
# 
# 
# Error:      6: dump-result "/"
#            "/"|D|0
#            "/rebel_zero"|F|0|0|"0"
#         end dump-result
#    
#    comparison of dump-results failed:
#       differing entry: /rebel_zero
# 
# 

# trace not accepted
# processing file 'example_traces/simple_dump_deviant-check.trace' failed
